pkgs/by-name/sp/spire/package.nix

```
@@ -38,11 +70,34 @@ buildGoModule rec {
     ln -vs $server/bin/spire-server $out/bin/spire-server
   '';
 
+  doInstallCheck = stdenv.buildPlatform.canExecute stdenv.hostPlatform;
```

```suggestion
  doInstallCheck = true;
```

`stdenv.buildPlatform.canExecute stdenv.hostPlatform` condition will be applied by stdenv